### ISSUE TITLE
Microsite logo link name

### ISSUE THREAD

**Issue by msayoung** [localgovdrupal/localgov_microsites_base#281]
We can upload a logo on a per microsite basis. 

This displays as a link to the homepage in the header as such: 

```
<div class="microsite-header__logo">
   <a href="/">
    <div class="field field--name-lgms-main-logo field--type-entity-reference field--label-hidden field__item">
    <img loading="lazy" src="/path/to/file" width="1912" height="303" alt="Alt text">
    </a>
</div>
```
This means the label of the link is inherited from the image alt text. 

The link text _should_ describe its purpose and the the image text _should_ describe the content of the image. 
There's quite a bit of variance in how this is handled elsewhere. Some use the site name, some use the logo text, some use the word home and some combine it. 

In the main LGD distro we seem to be using the Site name as the alt text for the logo. 

In summary. 

- If the site admin adds appropriate alt text to the image uploaded through the UI then there is no problem. However I often see logos given the alt text "Company name logo" which would be less great.
- Alternatively we could use some other text as the link / alt text in this case. eg `Site name` or `Site name: home`

Thoughts?

**Comment by markconroy** [comment]
I think we have 3 options

1. Use a `visually-hidden` span to set "home" as text. https://github.com/localgovdrupal/localgov_microsites_base/pull/286
2. Set the alt text for the logo to "home". https://github.com/localgovdrupal/localgov_microsites_base/pull/287
3. Add a new field called "Logo alt text" so we can have specific alt text for the logo if the media item is used anywhere else, and different alt text for when it is used as the header logo.

---
Thanks to [Big Blue Door](https://www.bigbluedoor.net/) for sponsoring my time to work on this.

**Comment by finnlewis** [comment]
@msayoung  could we have your thoughts on this? We want to come back to this https://github.com/localgovdrupal/localgov_microsites_base/pull/287

**Comment by msayoung** [comment]
The linked PR is setting the logo alt text to "home". This should be fine so long as that logo is never used without a link to the homepage. I can't remember off the top of my head whether it is used elsewhere. 

My slight preference would be to set it to "site_name : home" but I think this is good enough for now.

**Comment by markconroy** [comment]
@msayoung There are 2 PRs in the comment above, and also a 3rd option. We're trying to figure out which option is the preferred, or if another option should be considered.

From your comment, are you in favour of option 2?

**Comment by msayoung** [comment]
My personal preference is for option 1. 

From an accessibility point of view, both 1 or 2 have the same result .

**Comment by markconroy** [comment]
Thanks @msayoung, let's go with #286 so.
